### ISSUE TITLE
Summarizing kraken reports that use a custom taxonomy fails

### ISSUE THREAD

**Issue by Midnighter** [ewels/MultiQC#1305]
**Description of bug:**

I tried to summarize some kraken output using MultiQC. This has worked fine for me previously. I then generated some new analysis data using a custom database from EBI http://ftp.ebi.ac.uk/pub/databases/metagenomics/mgnify_genomes/human-gut/v1.0/uhgg_kraken2-db/ Trying to summarize these new kraken reports creates below error. I suspect that it is because of an ill-defined, custom taxonomy that is provided by EBI.

**MultiQC Error log:**

```python
============================================================
Module kraken raised an exception: Traceback (most recent call last):
  File "/home/moritz/.local/share/miniconda3/envs/kraken/lib/python3.8/site-packages/multiqc/multiqc.py", line 569, in run
    output = mod()
  File "/home/moritz/.local/share/miniconda3/envs/kraken/lib/python3.8/site-packages/multiqc/modules/kraken/kraken.py", line 62, in __init__
    self.general_stats_cols()
  File "/home/moritz/.local/share/miniconda3/envs/kraken/lib/python3.8/site-packages/multiqc/modules/kraken/kraken.py", line 166, in general_stats_cols
    top_one_hkey = '% {}'.format(top_five[0])
IndexError: list index out of range
============================================================
```

**File that triggers the error:**

**MultiQC run details (please complete the following):**
* Command used to run MultiQC: `multiqc .`
* MultiQC Version: 1.9
* Operating System:
```
Distributor ID: Ubuntu
Description:    Pop!_OS 18.04 LTS
Release:        18.04
Codename:       bionic
```
* Python Version: 3.8.3
* Method of MultiQC installation: conda (from conda-forge channel)

**Additional context**

http://ftp.ebi.ac.uk/pub/databases/metagenomics/mgnify_genomes/human-gut/v1.0/uhgg_kraken2-db/

**Comment by ewels** [comment]
Hi @Midnighter,

Thanks for reporting this. Could you please attach some output files from Kraken that trigger this error? I'll need them to fix the bug.

Phil

**Comment by aeu79** [comment]
Hi, @Midnighter, I ended up here with exactly the same error. 

Did you happen to use  --report-minimizer-data while generating the kraken reports? If that's the case, have a look to #1379.

Otherwise, the only thing I could think of is to check whether you have in the **Rank** column (4th) the taxa categories accepted by MultiQC (and kraken): UDKPCOFGS-:
 (U)nclassified, (R)oot, (D)omain, (K)ingdom, (P)hylum, (C)lass, (O)rder, (F)amily, (G)enus, or (S)pecies
(and S1, S2, G1, etc)

**Comment by Midnighter** [comment]
Oof, I had forgotten about this. I was not using `--report-minimizer-data`. I'll try to create an example soon™️

**Comment by Midnighter** [comment]
Hi @ewels, I tried again with the latest multiqc from conda. The error persists. I think it's due to the rank as @aeu79 describes. The ranks used are U, R, R1-7 so I guess that's not compatible. I've attached a sample report.

[ERR414351_0.51.k2report.gz](https://github.com/ewels/MultiQC/files/6253459/ERR414351_0.51.k2report.gz)

**Comment by ErikDanielsson** [comment]
This should be solved as of #1453.
